src/Settings/POLinesLimit.js

```diff
@@ -0,0 +1,139 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { Field } from 'redux-form';
+
+import SafeHTMLMessage from '@folio/react-intl-safe-html';
+import {
+  Callout,
+  Col,
+  Row,
+  TextField,
+} from '@folio/stripes/components';
+
+import POLinesLimitForm from './POLinesLimitForm';
+
+class POLinesLimit extends Component {
+  static manifest = Object.freeze({
+    recordId: {},
+    setPOLinesLimit: {
+      type: 'okapi',
+      records: 'configs',
+      path: 'configurations/entries?query=(module=ORDERS and configName=poLines-limit)',
+      POST: {
+        path: 'configurations/entries',
+      },
+      PUT: {
+        path: 'configurations/entries/%{recordId}',
+      },
+      throwErrors: false,
+    },
+  });
+
+  static propTypes = {
+    label: PropTypes.string.isRequired,
+    mutator: PropTypes.shape({
+      setPOLinesLimit: PropTypes.shape({
+        POST: PropTypes.func.isRequired,
+        PUT: PropTypes.func.isRequired,
+      }),
+      recordId: PropTypes.shape({
+        replace: PropTypes.func,
+      }),
+    }).isRequired,
+    resources: PropTypes.shape({
+      setPOLinesLimit: PropTypes.shape({
+        records: PropTypes.arrayOf(PropTypes.object),
+      }),
+    }),
+    stripes: PropTypes.shape({
+      intl: PropTypes.object.isRequired,
```

Comment:
it's anti-pattern, please check out docs here https://github.com/folio-org/stripes/blob/master/doc/i18n.md